pkg/plugin/keyvault.go

```diff
@@ -139,6 +184,41 @@ func (kvc *keyVaultClient) Decrypt(ctx context.Context, plain []byte) ([]byte, e
 	return bytes, nil
 }
 
+func (kvc *keyVaultClient) GetKeyID() string {
+	return kvc.keyID
+}
+
+func (kvc *keyVaultClient) ValidateAnnotations(annotations map[string][]byte) error {
+	if len(annotations) == 0 {
+		return fmt.Errorf("invalid annotations, annotations cannot be empty")
+	}
+
+	// validate key id
```

Comment:
```suggestion
	// validate key id
```

nit: these comments seem redundant, could we drop it?

Reply:
Readability... 😬